python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -1212,25 +1186,25 @@ def generate_ref_catalog(
             f"butler ingest-files -t direct {butler_root_path} cal_ref_cat refcats {sky_ecsv_file_name}"
         )
 
-    def ingest_data(self, butler_root_path: str, inst_name: str) -> None:
+    def ingest_data(self, butler_root_path: str, cam_type: CamType) -> None:
         """Ingest data into a gen3 data Butler.
 
         Parameters
         ----------
         butler_root_path : str
             Path to the butler repository.
-        inst_name : str
-            Instrument name.
+        cam_type : lsst.ts.wep.utils.CamType
+            Camera type.
         """
         output_img_dir = self.imsim_cmpt.output_img_dir
         files = " ".join(glob(os.path.join(output_img_dir, "amp*")))
 
-        if inst_name in ["lsst", "lsstfam", "comcam"]:
+        if cam_type in [CamType.LsstCam, CamType.LsstFamCam, CamType.ComCam]:
             runProgram(f"butler ingest-raws {butler_root_path} {files}")
 
-        if inst_name in ["lsst", "lsstfam"]:
+        if cam_type in [CamType.LsstCam, CamType.LsstFamCam]:
```

Comment:
I see we have these if statements many times. Would it make sense to have a function for that?

Reply:
Yes, added.